api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md

```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
 #include common-ref-eventparam
 
 ##### field(e.fromComponent): dxSortable | dxDraggable
-The instance of the item's source UI component.
+The instance of the item's source UI component. Works only for dragging between components.
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.itemData): any
-The dragged item's data. Note that you should specify the item data manually.
+The item's data before dragging. Note that you should specify the item data manually.
```

Comment:
See my previous comments regarding the -ing form. Can we restore the previous text?